hardware_interface/src/actuator.cpp

```diff
@@ -247,6 +247,7 @@ return_type Actuator::read(const rclcpp::Time & time, const rclcpp::Duration & p
     return return_type::OK;
   }
   if (
+    impl_->get_lifecycle_state().id() == lifecycle_msgs::msg::State::PRIMARY_STATE_UNKNOWN ||
     impl_->get_lifecycle_state().id() == lifecycle_msgs::msg::State::PRIMARY_STATE_UNCONFIGURED ||
     impl_->get_lifecycle_state().id() == lifecycle_msgs::msg::State::PRIMARY_STATE_FINALIZED)
```

Comment:
Might be a good idea to roll this into a method.

```cpp
constexpr bool lifecycleStateRequiresNoAction(const lifecycle_msgs::msg::State::_id_type state)
{
    return state == lifecycle_msgs::msg::State::PRIMARY_STATE_UNKNOWN ||
           state == lifecycle_msgs::msg::State::PRIMARY_STATE_UNCONFIGURED ||
           state == lifecycle_msgs::msg::State::PRIMARY_STATE_FINALIZED;
}

if (lifecycleStateRequiresNoAction(impl_->get_lifecycle_state().id()))
    return return_type::OK;
```

Reply:
Sure. I like the idea

Reply:
Done